source/functions/handle-errors.txt

```diff
@@ -0,0 +1,422 @@
+
+                  * - Collection Name
+                    - Name of your choosing (ex: ``failed_execution_logs``)
+
+                  * - Operation Type
+                    - Insert
+
+                  * - Full Document
+                    - Yes
+
+                  * - Document Preimage
+                    - No
+
+                  * - Select An Event Type
+                    - Function
+
+                  * - Function
+                    - Click :guilabel:`+ New Function`. Refer to the following information
+                      about the contents of the function.
+
+                  * - Advanced Configuration
+                    - No advanced configuration necessary.
+
+         .. tab::
+            :tabid: cli
+
+            Add configuration for the Database Trigger. For more information,
+            refer to the :ref:`Trigger configuration reference <appconfig-triggers>`.
+
+            .. literalinclude:: /examples/generated/functions/retryOperation.json
+               :language: json
+               :caption: triggers/retryOperation.json
+
+      Now add the code for the Function that the Trigger invokes.
+
+      The function ``retryOperation`` takes as a parameter ``logEntry``, the document that the
+      retry handler posted to the failed execution tracker collection.
+      Then ``retryOperation`` use :ref:`context.functions.execute() <context-functions>`
+      to invoke the main function with information from ``logEntry``.
+
+      .. tabs-realm-admin-interfaces::
+         :hidden: true
+
+         .. tab::
+            :tabid: ui
+
+            In the field :guilabel:`Function Name`, add ``retryOperationDbTrigger``.
+
+            For the field :guilabel:`Function`, add the following code:
+
+            .. literalinclude:: /examples/generated/functions/retryOperationDbTrigger.snippet.retry-operation-db-trigger-function.js
+               :language: js
+               :caption: functions/retryOperationDbTrigger.js
+
+         .. tab::
+            :tabid: cli
+
+            Add the Function metadata to :file:`functions/config.json`:
+
+            .. code-block:: js
+               :caption: functions/config.json
+
+               [
+                  // ...other configuration
+                  {
+                    "name": "retryOperationDbTrigger",
+                    "private": true
+                  }
+               ]
+
+            Add the following code to the file :file:`functions/retryOperationDbTrigger.js`:
+
+            .. literalinclude:: /examples/generated/functions/retryOperationDbTrigger.snippet.retry-operation-db-trigger-function.js
+               :language: js
+               :caption: retryOperationDbTrigger.js
+
+   .. step:: Write Function to retry
+
+      Now that you have the function handler and the retry Database Trigger Function,
+      you can write the main function.
+
+      In this example, the Function randomly throws an error when performing addition.
```

Comment:
When I saw "In this example" here, I assumed the code would be nearby, but it's actually quite far below the fold after some lengthy tables. Maybe we should say: "In the following example"?

Reply:
```suggestion
      In the following example, the Function randomly throws an error when performing addition.
```